### PR TITLE
Create san-diego-ca.md

### PR DESCRIPTION
**Pull request by playaj**
Here's the Create san-diego-ca.md

### CODE REVIEW

File: san-diego-ca.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+## Visiting San Diego CA
+
+- Bring $$$
```
Comment (githubteacher) [inline san-diego-ca.md:3]: To do what?

**Comment by githubteacher** [comment]
@playaj I am headed to San Diego soon, any good restaurant recommendations?

File: san-diego-ca.md (hunk 1 above)
Comment (playaj) [inline san-diego-ca.md:3]: To spend it. Its not too expensive, but you'll enjoy it.

**Comment by playaj** [comment]
@githubteacher Try Karl Strauss as a good restaurant

File: san-diego-ca.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+### No fun
+
+- Hard to beat it
+
+### Horrible weather
+
+- Not really bad
```
Comment (playaj) [inline san-diego-ca.md:11]: Except during El Niño

**Comment by playaj** [comment]
This closes #39

**Comment by playaj** [comment]
Not authorized to merge this pull request

**Pull request closed without merging**